src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -61,7 +70,23 @@ protected void doList(File path) throws FileNotFoundException, IOException {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 		List<SportSession> sessions = converter.listSportSessions(path);
 		for (SportSession session : sessions) {
-			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", type: " + session.getSportTypeId() + ", duration: " + session.getDuration() + "s");
+			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + new Time(session.getDuration()).toString());

```

Comment:
I get a weird result with `new Time(session.getDuration()).toString())` : my activities are displayed with +1 hour (e.g. 12 mins becomes 1:12:00). Working with dates and durations in Java is a bit of a pain, I'd suggest finding alternate solutions on StackOverflow.

Reply:
Fixed, use now: Duration.ofMillis(session.getDuration()).toString()